### DISCUSSION TITLE
syslog lack of time

### DISCUSSION THREAD

**Discussion by 18506711722** [Chia-Network/chia-blockchain#8558] [Ideas]
After adding syslog function,It facilitates cluster monitoring.but syslog lack of time

**Comment by BrandtH22** [comment]
@18506711722 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved.
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
